pipelines/_ingredients/cpFilterScan.yaml

```diff
@@ -0,0 +1,26 @@
+description: cp_pipe pipeline to measure filter scans.
+tasks:
+  cpFilterScanIsr:
+    class: lsst.ip.isr.isrTask.IsrTask
+    config:
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cpFilterScanIsrExp'
+      # This is copied from cpFlat.yaml:
+      doWrite: true
+      doDefect: true
+      doBrighterFatter: false
+      doFlat: true
+      doFringe: false
+      doApplyGains: false
+      doSaturation: false
+      maskNegativeVariance: false
+      doBias: true
+      doDark: true
+      doInterpolate: true
+      doSetBadRegions: true
+  cpFilterScanMerge:
+    class: lsst.cp.pipe.cpFilterScan.CpFilterScanTask
+    config:
+      connections.inputExpHandles: 'cpFilterScanIsrExp'
+      # This is still an intermediate product, although it's the end of this pipeline.
+      connections.outputData: 'cpFilterScan'
```

Comment:
I was going to comment that I don't like the name `cpFilterScan` as a final product but you say this is an intermediate product so 👍

Reply:
My thought is that these are results that can be converted into `transmission_filter` objects, likely using the monochromator scans as well, but that is beyond the scope of this task and ticket.